starter/assets/translations/fr.json

```diff
@@ -1,170 +1,220 @@
+      "UNSAVED_CHANGES.TITLE": "Modifications non enregistrées",
+      "UNSAVED_CHANGES.TEXT": "Voulez-vous vraiment quitter sans enregistrer les modifications?",
+      "UNSAVED_CHANGES.ARIA_LABEL": "confirmation des modifications non enregistrées",
+      "UNSAVED_CHANGES.CONFIRM": "Confirmer",
+      "UNSAVED_CHANGES.CANCEL": "Ne quittez pas"
+    },
+
+    "DROPDOWN": {
+      "SELECTED": "{{count}} sélectionné",
+      "SELECTED_PLURAL": "{{count}} sélectionnés"
+    },
+
+    "FILE_UPLOAD": {
+      "NAME": "Nom",
+      "SIZE": "Taille",
+      "TYPE": "Type",
+      "UPLOAD_QUEUE": "File d'envoi",
+      "START": "Démarrer l'envoi",
+      "PROGRESS": "Progression",
+      "DRAG_DROP": "Drag & drop les fichiers ici ",
+      "OR": "ou ",
+      "SELECT_FILES": "selectioner les fichiers"
+    },
+
+    "GRID": {
+      "SHOW_MORE": "Montrer plus..."
+    },
+
+    "ICONS": {
+      "MENU": "Menu",
+      "TOGGLE_MODE": "Mode compact/plein",
+      "APP_DATA": "Données d'application",
+      "ADD_ITEM": "Ajouter",
+      "EDIT_ITEM": "Modifier",
+      "DELETE_ITEM": "Supprimer",
+      "APPROVE_ITEM": "Approuver",
+      "REJECT_ITEM": "Rejeter",
+      "RELOAD_PAGE": "Recharger la page",
+      "OPEN_SELECTION": "Ouvrir la sélection",
+      "SAVE_ITEM": "Sauver",
+      "CLOSE_ITEM": "Fermer",
+      "RESET": "Réinitialiser",
+      "SEARCH": "Chercher",
+      "NEW_ITEM": "Nouveau",
+      "SAVE_AND_NEXT": "Sauver et Suivant"
+    },
+
+    "LOGIN": {
+      "TITLE": "Connexion",
+      "USERNAME": "Nom d'utilisateur",
+      "PASSWORD": "Mot de passe",
+      "PASSWORD_FORGOTTEN": "J'ai oublié mon mot de passe",
+      "LOGIN_BUTTON": "Connexion",
+      "USERS_LIST": "Utilisateurs disponibles",
+      "NO_PROFILE": "Aucun profil utilisateur disponible"
+    },
+
+    "MESSAGE_PANE": {
+      "ERROR_MESSAGES": "Messages d'erreur",
+      "WARNING_MESSAGES": "Messages d'avertissement",
+      "INFO_MESSAGES": "Messages d'information",
+      "EXPAND": "Développer le volet des messages",
+      "DISMISS_ALL": "Supprimer tous les messages",
+      "COLLAPSE": "Réduire le volet des messages",
+      "CLOSE_MESSAGE": "Supprimer ce message"
+    },
+
+    "MULTI_COLUMN_SORTING": {
+      "TITLE": "Tri multi-colonnes",
+      "ADD_SORTING_LEVEL": "Ajouter le niveau de tri",
+      "REMOVE_SORTING_LEVEL": "Supprimer le niveau de tri",
+      "SORT_BY": "Trier par",
+      "THEN_BY": "Puis par",
+      "CANCEL": "Annuler",
+      "SAVE": "Enregistrer"
+    },
+
+    "PAGINATION": {
+      "PAGE": "Page",
+      "ITEMS_PER_PAGE": "Eléments par page"
+    },
+
+    "PRELOADING": {
+      "FETCHING_USER_PROFILE": "Initialisation...",
+      "FETCHING_USER_PROFILE_FAILURE": "Échec de l'initialisation: impossible de récupérer le profil de l'utilisateur.",
+      "CONTACT_IT_SUPPORT": "Veuillez contacter votre service d'assistance informatique.",
+      "CORRELATION_ID": "ID de corrélation",
+      "RELOAD": "Recharger"
+    },
+
+    "RBAC": {
```

Comment:
The RBAC translations will have to be moved later on to their own module.

Reply:
All translation need to be moved to their corresponding modules when we create them in stark-ui.